### PR TITLE
codelive/trygger

### PR DESCRIPTION
**Pull request by codefriar**
### What does this PR do?

Adds Metadata driven triggers and associated test recipes.

### CODE REVIEW

File: force-app/main/default/classes/MDTSecondAccountTriggerHandler.cls-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>50.0</apiVersion>
```
Comment (msrivastav13) [inline force-app/main/default/classes/MDTSecondAccountTriggerHandler.cls-meta.xml:3]: I think we should bump this to `51.0`

Comment (codefriar) [reply force-app/main/default/classes/MDTSecondAccountTriggerHandler.cls-meta.xml:3]: good catch. fixing.

**Comment by codefriar** [comment]
> good catch. fixing. 

Fixed.

File: force-app/main/default/classes/Trigger Recipes/sampleHandler.cls [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * @description This class is a sample trigger handler for use while testing
+ * the metadataTriggerHandler. Because custom metadata cannot be inserted, and
+ * because the MetadataTriggerHandler instantiates handler classes from custom
+ * metadata records, even when we stub/mock the metadata record retrieval we
+ * still need an actuall class that it can instantiate.
+ *
+ * Note, this class is annotated with @isTest to prevent it's use outside of
+ * tests, not because it contains tests.
+ */
+public with sharing class sampleHandler extends TriggerHandler {
```
Comment (msrivastav13) [inline force-app/main/default/classes/Trigger Recipes/sampleHandler.cls:12]: Maybe for consistency, we change it to `SampleHandler `!

File: package.json [hunk 3]
```diff
@@ -40,5 +40,8 @@
         "prettier": "^2.2.1",
         "prettier-plugin-apex": "^1.8.0",
         "semver": "^7.3.4"
+    },
+    "dependencies": {
```
Comment (msrivastav13) [inline package.json:44]: @codefriar was this intentional? I defer this to @pozil as I am not sure if this is needed!

**Review by msrivastav13 — COMMENTED** [review]
Looks good to me except for Salesforce CLI npm dependency!

I will defer to @pozil on that!

File: package.json (hunk 3 above)
Comment (pozil) [reply package.json]: Thanks for removing it @codefriar. Is the `package-lock.json` up to date with that change?

**Review by pozil — APPROVED** [review]
Looking good. Can you run prettier and address the open items before merging?

**Pull request merged**